hdfs-executor/src/main/java/org/apache/mesos/hdfs/executor/NameNodeExecutor.java

```diff
@@ -155,14 +156,16 @@ private void initNameNode(ExecutorDriver driver, String dnsName) throws Exceptio
         // is used as a mutex indicates whether or not any NameNode has ever been formatted.
         // The first NameNode to acquire the mutex and find that no NameNode has ever been
         // formatted, formats itself.  All others bootstrap.
```

Comment:
@elingg 
Let me explain. The logic of initializing is following:
- if first run, then format;
- if formatted, then bootstrap;
- if no backup, then bootstrap even if already bootstrapped;
- if backup exists and bootstrapped, do nothing and let NN recover from backupDir;

Here I am creating backup for both NNs.